webapp/Connector/src/view/SingleAxisExplorer.js

```diff
@@ -191,6 +191,64 @@ Ext.define('Connector.view.SingleAxisExplorer', {
         }
     },
 
+    /**
+     * Render an optional tooltip on the explorer bar
+     */
+    onMouseEnterExplore : function(view, rec) {
+        var name = this.parseExplorerBarName(rec.data.hierarchy, rec.data.uniqueName);
+        if (name) {
+            var highlighted = Ext.dom.Query.select('span.barlabel[uniquename="' + rec.data.uniqueName + '"]');
+            var el = Ext.get(highlighted[0]);
+
+            if (el) {
+                var calloutMgr = hopscotch.getCalloutManager();
+                var id = el.id;
+                var tooltip = this.getTooltip(rec.data.hierarchy, name);
+
+                if (tooltip) {
+                    var displayTooltip = setTimeout(function() {
+                        calloutMgr.createCallout(Ext.apply({
+                            id: id,
+                            arrowOffset: 20,
+                            yOffset: 0,
+                            showCloseButton: false,
+                            target: highlighted[0],
+                            placement: 'top',
```

Comment:
Usually i've seen the placement of the tooltip on the right (or left), in this case, shouldn't it be on the right? If so, then  xOffset & yOffset will need to be updated as well

Reply:
For the explorer bars it was difficult to place it precisely to the right of the bar text so that it wouldn't obscure the text, it was easier and more consistent to place it over the bar. Also if we tried to place it to the right, depending on the length of the text, the tooltip position would jump around which seemed sub optimal. I think normally we would place it to the left but since that wasn't possible above was the next best option.